jujugui/static/gui/src/app/components/sharing/_sharing.scss

```diff
@@ -30,13 +34,15 @@
     &-access {
       @extend .vertical-center;
       position: absolute;
-      right: 0;
-      text-transform: capitalize;
+      right: 1px;  // Provide room for scrollbars on longer lists.
```

Comment:
I'm not sure about this whole 'position: absolute; right: 1px;' set up but it can be reviewed in a follow up.